docs/dietpi_tools.md

```diff
@@ -192,11 +192,26 @@ When you execute the certificate installation it also installs it for your selec
     To be accessible from the internet, typically your router needs a port forwarding configuration to route incoming HTTP and HTTPS accesses to your DietPi system.  
     Although you only need a HTTPS protocol forwarding (typically port 433), you also need to forward the HTTP protocol (typically port 80) to your DietPi system, otherwise the certification renewal procedure will fail (due to the fact that the certification renewal procedure takes place several months later you may have forgotten this issue).
 
-### DietPi NordVPN
+### DietPi VPN
 
-Run `dietpi-nordvpn`.
+DietPi-VPN is a combination of OpenVPN installation and DietPi front end GUI. Allowing all VPN users to quickly and easily connect to any NordVPN, ProtonVPN, or any other server that uses OpenVPN in TCP or UDP, using only open source software.
 
-![DietPi-NordVPN screenshot](assets/images/dietpi-nordvpn.jpg){: width="642" height="207" loading="lazy"}
+![DietPi-VPN screenshot](assets/images/dietpi-vpn.jpg){: width="642" height="300" loading="lazy"}
+=== "Requires VPN Subscription"
```

Comment:
There is a needed empty line between lines 199 and 200, otherwise the image "dietpi-vpn.jpg" does not occur.

**Without empty line:**

![grafik](https://user-images.githubusercontent.com/57687516/112805909-7ea1b680-9076-11eb-960a-7d692b233bb2.png)

**With empty line:**

![grafik](https://user-images.githubusercontent.com/57687516/112806043-a4c75680-9076-11eb-9602-35b61ca18912.png)